core/arch/x86/mangle.c

```diff
@@ -390,7 +390,9 @@ insert_push_all_registers(dcontext_t *dcontext, clean_call_info_t *cci,
             pre_avx512_push = INSTR_CREATE_label(dcontext);
             PRE(ilist, instr,
                 INSTR_CREATE_cmp(dcontext,
-                                 OPND_CREATE_ABSMEM(d_r_avx512_code_in_use, OPSZ_1),
+                                 OPND_CREATE_ABSMEM(vmcode_get_executable_addr(
```

Comment:
You merged in master?  Please wait until the final LGTM: as discussed before, this pollutes the diff view vs what I've already reviewed.

Reply:
It was actually more "accidental", I remembered it but it was too late.